Modules/Sources/WordPressUI/Views/Settings/Experimental Features/ExperimentalFeaturesList.swift

```diff
@@ -0,0 +1,55 @@
+import SwiftUI
+
+public struct ExperimentalFeaturesList: View {
+
+    @ObservedObject
+    var viewModel: ExperimentalFeaturesViewModel
+
+    package init(viewModel: ExperimentalFeaturesViewModel) {
+        self.viewModel = viewModel
+    }
+
+    public var body: some View {
+        List(viewModel.items) { item in
+            HStack {
+                Toggle(item.name, isOn: Binding<Bool>(
+                    get: {
+                        viewModel.dataProvider.value(for: item)
+                    },
+                    set: { newValue in
+                        viewModel.dataProvider.didChangeValue(for: item, to: newValue)
```

Comment:
I tested it and it seems to work, but it I'm not sure why because a custom binding requires an observable source of truth. In this case, the view state doesn't change when the `didChangeValue:` is called. I don't remember what you need to do to break, but, just to be safe, I suggest moving this to the view model and adding `objectWillChange.send()` call.

```swift
final class ExperimentalFeaturesViewModel {
    func binding(for item: Item) -> Binding<Boo> {
        Binding<Bool>(
                     get: {
                         dataProvider.value(for: item)
                     },
                     set: { newValue in
                         self.objectWillChange.send() // <---
                         dataProvider.didChangeValue(for: item, to: newValue)
                     }
        )
    }
}
```

Reply:
Sure – that's nice and clean – done in 48ced541602700f10f282c875deedf18281d8ad3